### PR TITLE
Remove "right click" from the strings

### PR DESCRIPTION
**Pull request by apple502j**
This replaces "right click" with "open context menu" (or similar words).

### CODE REVIEW

File: addons-l10n/en/editor-devtools.json [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
   "editor-devtools/help-new": "★New★",
   "editor-devtools/help-make-space": "Make Space",
-  "editor-devtools/help-make-space-desc": "Make extra space around a script by right clicking on the code stack and choosing \"Make Space\".",
+  "editor-devtools/help-make-space-desc": "Make extra space around a script choosing \"Make Space\" on the block context menu.",
   "editor-devtools/copy-to-clipboard": "Copy to Clipboard",
-  "editor-devtools/copy-to-clipboard-desc": "Right click a block and 3 new options are available to Copy All, Copy Block, and Cut Block. The Copy All will copy to the clipboard everything including and below the block you clicked on. Copy block will only copy the current block and its contents, but nothing below. And cut block will copy it and remove it from the workspace.",
+  "editor-devtools/copy-to-clipboard-desc": "Block context menu includes 3 new options: Copy All, Copy Block, and Cut Block. The Copy All will copy to the clipboard everything including and below the block you clicked on. Copy block will only copy the current block and its contents, but nothing below. And cut block will copy it and remove it from the workspace.",
```
Comment (RedGuy12) [inline addons-l10n/en/editor-devtools.json:15]:
```suggestion
  "editor-devtools/copy-to-clipboard-desc": "Block context menus includes 3 new options: Copy All, Copy Block, and Cut Block. The Copy All will copy to the clipboard everything including and below the block you clicked on. Copy block will only copy the current block and its contents, but nothing below. And cut block will copy it and remove it from the workspace.",
```

File: addons-l10n/en/editor-devtools.json [hunk 2]
```diff
@@ -7,16 +7,16 @@
...
   "editor-devtools/improved-tidy-up": "Improved Code Tidy Up",
-  "editor-devtools/improved-tidy-up-desc": "Right click on the scripts window to pop up the menu and the Clean up Blocks option will have been replaced by a Clean up Blocks + option. Use this to tidy your scripts and it will preserve your scripts columns as well as attempt to align the comments and remove all those orphaned variables, etc.",
+  "editor-devtools/improved-tidy-up-desc": "\"Clean up Blocks\" option on the block context menu is replaced with \"Clean up Blocks +\" option. Use this to tidy your scripts and it will preserve your scripts columns as well as attempt to align the comments and remove all those orphaned variables, etc.",
   "editor-devtools/help-new": "★New★",
   "editor-devtools/help-make-space": "Make Space",
-  "editor-devtools/help-make-space-desc": "Make extra space around a script by right clicking on the code stack and choosing \"Make Space\".",
+  "editor-devtools/help-make-space-desc": "Make extra space around a script choosing \"Make Space\" on the block context menu.",
```
Comment (RedGuy12) [inline addons-l10n/en/editor-devtools.json:13]:
```suggestion
  "editor-devtools/help-make-space-desc": "Make extra space around a script choosing \"Make Space\" in the block context menu.",
```

File: addons-l10n/en/editor-devtools.json [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
   "editor-devtools/swap-variable": "Swap Variable in Sprite",
-  "editor-devtools/swap-variable-desc": "Right click a variable in your scripts for this new option. It allows you to switch all references to this variable in the current sprite all in one go to another variable. This is great for when you made a mistake and want to switch from one variable to another or need to change from a 'for all sprites' to a 'for this sprite only'. This option will not remove the old variable and will not affect any other sprites variables.",
+  "editor-devtools/swap-variable-desc": "This new option is available on the variable context menu. It allows you to switch all references to this variable in the current sprite all in one go to another variable. This is great for when you made a mistake and want to switch from one variable to another or need to change from a 'for all sprites' to a 'for this sprite only'. This option will not remove the old variable and will not affect any other sprites variables.",
```
Comment (RedGuy12) [inline addons-l10n/en/editor-devtools.json:19]:
```suggestion
  "editor-devtools/swap-variable-desc": "This new option is available in the variable context menu. It allows you to switch all references to this variable in the current sprite all in one go to another variable. This is great for when you made a mistake and want to switch from one variable to another or need to change from a 'for all sprites' to a 'for this sprite only'. This option will not remove the old variable and will not affect any other sprites variables.",
```

File: addons/bitmap-copy/addon.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "info": [
     {
       "type": "notice",
-      "text": "\"Right click \u2192 copy\" is not supported. You must press Ctrl+C while a bitmap image is selected.",
+      "text": "Context menus are not supported. You must press Ctrl+C while a bitmap image is selected.",
```
Comment (RedGuy12) [inline addons/bitmap-copy/addon.json:7]:
```suggestion
      "text": "Clicking \"copy\" on context menus is not supported. You must press Ctrl+C while a bitmap image is selected.",

File: addons/blocks2image/addon.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Save blocks as image",
-  "description": "Right click the editor workspace to export blocks as SVG/PNG images.",
+  "description": "Add options to export blocks as SVG/PNG images on the editor context menu.",
```
Comment (RedGuy12) [inline addons/blocks2image/addon.json:3]:
```suggestion
  "description": "Add options to export blocks as SVG/PNG images to the editor context menu.",
```

File: addons/folders/addon.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Sprite folders",
-  "description": "Adds folders to sprite lists, as well as costume and sound lists. To create a folder, right click any sprite and click \"create folder\". Click a folder to open or close it. Right click a sprite to see what folders you can move it to, or alternatively drag and drop it into an open folder.",
+  "description": "Adds folders to sprite lists, as well as costume and sound lists. To create a folder, open the context menu on any sprite and click \"create folder\". Click a folder to open or close it. Open the context menu on a sprite to see what folders you can move it to, or alternatively drag and drop it into an open folder.",
```
Comment (RedGuy12) [inline addons/folders/addon.json:3]:
```suggestion
  "description": "Adds folders to sprite lists, as well as costume and sound lists. To create a folder, open a sprite's context menu and select \"create folder\". Click a folder to open or close it. Open a sprite's context menu on a sprite to see what folders you can move it to, or alternatively drag and drop it into an open folder.",
```

**Review by RedGuy12 — CHANGES_REQUESTED** [review]

File: addons-l10n/en/editor-devtools.json (hunk 1 above)
Comment (apple502j) [reply addons-l10n/en/editor-devtools.json:15]: This is English, we can't just change a word.

Comment (RedGuy12) [reply addons-l10n/en/editor-devtools.json:15]: oh right how bout now

**Comment by WorldLanguages** [comment]
I'm not sure about this. If merged it will be for v1.15.0

**Pull request closed without merging**